src/lib/grammar.rs

```diff
@@ -386,4 +389,29 @@ mod test {
         assert_eq!(grm.prod_precs[6].unwrap(), Precedence{level: 4, kind: AssocKind::Nonassoc});
         assert!(grm.prod_precs[7].is_none());
     }
+
+    #[test]
+    fn test_prec_override() {
+        let grm = Grammar::new(&parse_yacc(&"
+            %start expr
+            %left '+' '-'
+            %left '*' '/'
+            %%
+            expr : expr '+' expr
+                 | expr '-' expr
+                 | expr '*' expr
+                 | expr '/' expr
+                 | %prec '-'  expr '*'
```

Comment:
It's just how I understood the following sentence from the yacc guide:

> %prec appears immediately after the body of the grammar rule, before the action or closing semicolon, and is followed by a token name or literal. It causes the precedence of the grammar rule to become that of the following token

So the formula is: `PROD %prec TOKEN;`, where TOKEN is the precedence we want for PROD.